app/models/generic_object_definition.rb

```diff
@@ -138,7 +138,7 @@ def delete_property_association(name)
 
   def add_property_method(name)
     properties[:methods] << name.to_s unless properties[:methods].include?(name.to_s)
```

Comment:
what about something like this?

```suggestion
    unless properties[:methods].include?(name.to_s)
      properties[:methods] << name.to_s
      save!
    end
```